examples/complete/main.tf

```diff
@@ -29,15 +29,15 @@ module "managed_rules" {
   ]
 
   redshift_cluster_maintenancesettings_check_parameters = {
-    allowVersionUpgrade = "true"
+    allowVersionUpgrade = true
   }
 
   rule_overrides = {
     acm-certificate-expiration-check = {
       description = "Checks if AWS Certificate Manager Certificates in your account..."
 
       input_parameters = {
-        daysToExpiration = tostring(var.acm_certificate_expiration_check)
```

Comment:
The same treatment as above was made here. The `daysToExpiration` parameter in this example expects a `number` that will be converted to a string later.